src/components/bookmarks/BookmarkSection.tsx

```diff
@@ -84,27 +64,17 @@ export const BookmarkSection = ({
       });
   }, [data, bookmarkKey, loading, setConnectionState]);
 
-  if (loading) {
-    return (
-      <LoadingContainer>
-        <ActivityIndicator color={colors.accent} />
-      </LoadingContainer>
-    );
-  }
-
-  if (!data) return null;
-
-  const listData = parseListItemsFromQuery(query, data, ids.length > 3, categoryTitleDetail);
-
   return (
-    <View>
-      <SectionHeader title={sectionTitle || getTitle(query)} onPress={onPressShowMore} />
-      <ListComponent data={listData} navigation={navigation} query={query} />
-      {ids.length > 3 ? (
-        <Wrapper>
-          <Button title={texts.bookmarks.showAll} onPress={onPressShowMore} />
-        </Wrapper>
-      ) : null}
-    </View>
+    <DataListSection
+      loading={loading}
+      navigation={navigation}
+      query={query}
+      buttonTitle={texts.bookmarks.showAll}
+      navigate={onPressShowMore}
+      sectionData={data}
+      sectionTitleDetail={categoryTitleDetail}
+      sectionTitle={sectionTitle}
+      showButton={ids.length > 3}
```

Comment:
props here could be ordered the same way as in `HomeSection`

```
<DataListSection
  loading={loading}
  navigation={navigation}
  query={query}
  buttonTitle={buttonTitle}
  sectionData={data}
  navigate={navigate}
  sectionTitle={title}
  sectionTitleDetail={titleDetail}
  showButton
/>
```

or other way around